packages/wdio-testingbot-service/src/launcher.ts

```diff
@@ -1,10 +1,10 @@
 import { performance, PerformanceObserver } from 'perf_hooks'
 import { promisify } from 'util'
 
-import testingbotTunnel from 'testingbot-tunnel-launcher'
+import testingbotTunnel = require('testingbot-tunnel-launcher')
```

Comment:
Why should we switch to commonjs import just for this package?

Reply:
As specified in the docs, it's the only way to work with `export =`
https://www.typescriptlang.org/docs/handbook/modules.html#export--and-import--require

Reply:
Why do we make the change in `testingbot-tunnel-launcher.d.ts` then as well as in the mock? The [original package](https://github.com/testingbot/testingbot-tunnel-launcher/blob/master/index.js) has a normal commonjs export:

```js
module.exports = require('./build/tunnel-launcher')
```

which can be imported using

```js
import TunnelLauncher from 'testingbot-tunnel-launcher'
```

So we should try to mimic this in our TS files and mocks.